e2e/test/manifest/linode-blockstorage-csi-driver.yaml

```diff
@@ -0,0 +1,560 @@
+# pkg/linode-bs/deploy/kubernetes/01-csi-nodeinfo.yaml
```

Comment:
Based on `test` depending upon `test-manifest` Makefile, this file should not be committed, and it should be gitignored in this directory. Maybe you committed it by mistake?

Reply:
It was committed in the original PR, so I left it. I'll remove it though.